### ISSUE TITLE
Syntax error using Python 3.12.1

### ISSUE THREAD

**Issue by RDHWalker** [bkerler/netgear_telnet#40]
Syntax error as shown below under Python version 3.12.1:
```
C:\Users\rdhw>python \local\bin\telnet-enable.py 192.168.XXX.1 C8:9E:43:XX:XX:XX admin XXXXXXXXXX
C:\local\bin\telnet-enable.py:50: SyntaxWarning: invalid escape sequence '\s'
  for v in (re.split('\s+', i) for i in arp_data_raw))
C:\local\bin\telnet-enable.py:660: SyntaxWarning: invalid escape sequence '\s'
  for line in re.findall('([-.0-9]+)\s+([-0-9a-f]{17})\s+(\w+)', data):
Netgear Telnet enabler V3.1 (c) B.Kerler 2021-2023
Done sending pw data XXXXXXXXXX to 192.168.XXX.1:23
```

**Comment by Iliasbrv** [comment]
I get the same error on my RAX70 with Firmware Version 1.0.1.68

**Comment by sshirey76** [comment]
You should be able to address this by editing your 49-50 
From: 
    parsed_arp_table = (dict(zip(('ip_address', 'type', 'flags', 'hw_address', 'mask', 'device'), v))
                        for v in (re.split('\s+', i) for i in arp_data_raw))
To:
    parsed_arp_table = [dict(zip(('ip_address', 'type', 'flags', 'hw_address', 'mask', 'device'), v)) 
    			for v in (re.split(r'\s+', i) for i in arp_data_raw)]

For the referenced at your 660: 
From: 
                for line in re.findall('([-.0-9]+)\s+([-0-9a-f]{17})\s+(\w+)', data):
To:
                for line in re.findall(r'([\[-.0-9\]]+)\s+([-0-9a-f]{17})\s+(\w+)', data):

They are caused by the way the backslash character \ is used in the regular expression patterns. 
Example on the 2nd error.. 
In the pattern '(\[-.0-9\]+)\\s+(\[-0-9a-f\]{17})\\s+(\\w+)', the \\s is interpreted as an invalid escape sequence because the backslash \ is not being treated as a literal character.
To fix this issue, you use a raw string literal by prefixing the pattern with r. This way, backslashes \ are treated as literal characters, and you don't need to escape them.  Hope it helps someone!

**Comment by Pawel-B7** [comment]
@sshirey76
Thanks for your help!
Unfortunately the script still doesn't start telnet
I'm checking on Orbi RBR40 v2.7.3.22
The script is executed and the window closes
D:\>telnet-enable.py 10.0.0.1 A0:40:A0:7C:54:4E admin Test1234
Netgear Telnet enabler V3.1 (c) B.Kerler 2021-2023
What message should be displayed after the script has been successfully executed?

**Comment by bkerler** [comment]
Fixed in PR #47

**Comment by Pawel-B7** [comment]
Doesn't work after reboot :(
